central/declarativeconfig/health/datastore_impl.go

```diff
@@ -0,0 +1,75 @@
+package health
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/declarativeconfig/health/store"
+	"github.com/stackrox/rox/central/role/resources"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+)
+
+var (
+	integrationSAC = sac.ForResource(resources.Integration)
```

Comment:
General question on the SAC checks.  SAC is also checked in the store.  Are the checks the same or are the checks here at a different level than the ones in the store?  Basically I'm just asking if we are checking the same SAC twice.

Reply:
That's a very good observation - I removed duplicating checks from `datastore_impl`